### PR TITLE
add QE perf profiles: aws-perfscale-qe and aws-perfscale-lrc-qe

### PR DESCRIPTION
**Pull request by yunjiang29**
ci tools PR: https://github.com/openshift/ci-tools/pull/3537

### CODE REVIEW

File: core-services/prow/02_config/generate-boskos.py [hunk 1]
```diff
@@ -73,6 +73,12 @@
     'aws-perf-qe-quota-slice': {
         'us-west-2': 3,
     },
+    'aws-perfscale-qe-quota-slice': {
+        'us-west-2': 3,
```
Comment (svetsa-rh) [inline core-services/prow/02_config/generate-boskos.py:77]: @yunjiang29 This quota will be too small for us. As we have several Self Managed, Classic Rosa and Hypershift cron jobs and also multiple team members actively working on adding more tests and analyzing results/debugging etc.. 

Can we set this to 'us-west-2': 15 at least?

File: core-services/prow/02_config/generate-boskos.py [hunk 2]
```diff
@@ -73,6 +73,12 @@
...
+    },
+    'aws-perfscale-lrc-qe-quota-slice': {
+        'us-west-2': 3,
```
Comment (svetsa-rh) [inline core-services/prow/02_config/generate-boskos.py:80]: This one we don't need as much but slightly more. 
Can we set it to:
'us-west-2': 5

**Review by svetsa-rh — CHANGES_REQUESTED** [review]

File: core-services/prow/02_config/generate-boskos.py (hunk 1 above)
Comment (yunjiang29) [reply core-services/prow/02_config/generate-boskos.py:77]: @svetsa-rh  This number means the **concurrent** job running in the CI. Even for aws-qe profile, 40 is enough for all of our tests.
I'd suggest to set to 5, and see if it works. WDYT?

Comment (svetsa-rh) [reply core-services/prow/02_config/generate-boskos.py:77]: @yunjiang29 Even for aws-qe I have hit lease issues and quota issues several times last 2-3 weeks. `'aws-perfscale-qe-quota-slice'` is the new profile we will be using for majority of tests.
I think 5 will be little too less. May be we can start with 10 and see if we hit any issues.

File: core-services/prow/02_config/generate-boskos.py (hunk 2 above)
Comment (svetsa-rh) [reply core-services/prow/02_config/generate-boskos.py:80]: Yeah, even when using `aws-qe` I have hit lease issues and scaling issues for Rosa and HCP clusters.

File: core-services/prow/02_config/generate-boskos.py (hunk 1 above)
Comment (yunjiang29) [reply core-services/prow/02_config/generate-boskos.py:77]: @svetsa-rh you are using `aws-qe` for your testing, and you will switch to `aws-perfscale-qe-quota-slice` after it's ready., is it right?

File: core-services/prow/02_config/generate-boskos.py (hunk 2 above)
Comment (yunjiang29) [reply core-services/prow/02_config/generate-boskos.py]: setting `aws-perfscale-lrc-qe-quota-slice` to 5, @qiliRedHat is it OK for lrc account?

Comment (qiliRedHat) [reply core-services/prow/02_config/generate-boskos.py]: @yunjiang29 I'm ok with 5 for lrc account.

**Comment by yunjiang29** [comment]
@bbguimaraes can you approve this PR? thanks

File: core-services/prow/02_config/generate-boskos.py (hunk 1 above)
Comment (svetsa-rh) [reply core-services/prow/02_config/generate-boskos.py]: correct @yunjiang29 - for Classic Rosa and HCP, we will switch right away.

Comment (svetsa-rh) [reply core-services/prow/02_config/generate-boskos.py]: correct @yunjiang29 - for Classic Rosa and HCP, we will switch right away.

**Comment by svetsa-rh** [comment]
@bbguimaraes Could you ptal?

**Pull request merged**